### DISCUSSION TITLE
Red battery symbol

### DISCUSSION THREAD

**Discussion by mkrezdo** [devemux86/cruiser#223] [Q&A] [answered]
Yesterday it showed me during the navagion a red battery symbol (show battery status is on). Normally it's colored same than all the other fields.
Then I stoped charging and started again and it was normal again.

-> What does this mean? (I'm not sure but can it be that it shows that the battery is charging but nevertheless the level will reduce because it's only charging very little?)

Marco

**Comment by devemux86** [comment, marked as answer]
Hi @mkrezdo,

Red battery alert should appear when:
- Battery level drops below 20%
- Battery level drops _while phone is plugged_

Did any of this happen?

Please see also the relevant discussion here:
- https://github.com/devemux86/cruiser/discussions/130#discussioncomment-5715837

**Reply by mkrezdo** [reply]
Yes - I guess the seconed point could happen in this moment. It was very hot - perhaps because of this the inductive charge reduced the power (it was still charging). Battery level was around 80%.
I switched then from iductive to cabel and the red symbol was away.
